src/screens/wallet/screen/steemView.js

```diff
@@ -25,36 +25,36 @@ const SteeemView = ({ handleOnSelected, index, currentIndex }) => (
             savingSteemDropdown,
             navigate,
           }) => (
-            <WalletHeader
-              componentDidUpdate={() => handleOnSelected(transferHistory, isLoading)}
-              index={index}
-              claim={claimRewardBalance}
-              fetchUserActivity={handleOnWalletRefresh}
-              isClaiming={isClaiming}
-              isLoading={isLoading}
-              refreshing={refreshing}
-              unclaimedBalance={0}
-              userBalance={[
-                { balance: steemBalance, nameKey: 'steem', options: steemDropdown },
-                { balance: steemSavingBalance, nameKey: 'saving', options: savingSteemDropdown },
-              ]}
-              handleOnDropdownSelected={option => navigate(option, 'STEEM')}
-              type="steem"
-              currentIndex={currentIndex}
-              showIconList={false}
-              valueDescriptions={[
-                {
-                  textKey: 'estimated_value',
-                  value: <FormatedCurrency isApproximate value={estimatedValue} />,
-                  subTextKey: 'estimated_value_desc',
-                },
-              ]}
-            />
-          )}
+              <WalletHeader
+                componentDidUpdate={() => handleOnSelected(transferHistory, isLoading)}
+                index={index}
+                claim={claimRewardBalance}
+                fetchUserActivity={handleOnWalletRefresh}
+                isClaiming={isClaiming}
+                isLoading={isLoading}
+                refreshing={refreshing}
+                unclaimedBalance={0}
+                userBalance={[
+                  { balance: steemBalance, nameKey: 'steem', options: steemDropdown },
+                  { balance: steemSavingBalance, nameKey: 'savingsteem', options: savingSteemDropdown },
+                ]}
+                handleOnDropdownSelected={option => navigate(option, 'STEEM')}
+                type="steem"
+                currentIndex={currentIndex}
+                showIconList={false}
```

Comment:
has wrong spaces all entire file

Reply:
I ran `yarn run lint:fix`, it should be fixed now.